compaction/compaction_strategy.cc

```diff
@@ -765,6 +765,13 @@ compaction_strategy make_compaction_strategy(compaction_strategy_type strategy,
     case compaction_strategy_type::incremental:
         impl = make_shared<incremental_compaction_strategy>(incremental_compaction_strategy(options));
         break;
+    case compaction_strategy_type::unsupported_in_memory:
```

Comment:
This is not a direct port and will cause unnecessary conflicts in the next merge.

Make it a straightforward port without gratuitous changes, either by editing the patch or changing enterprise.

Reply:
ok, let's do the straight forward port first and follow-up with changes